src/components/BlogPostFooter.jsx

```diff
@@ -0,0 +1,43 @@
+import React from "react";
+
+import STBannerBackdrop from "../images/supertokens-logo-backdrop.png";
+
+const BlogPostFooter = ({ idSlug }) => {
+
+  return (
+    <footer id={`last_section_${idSlug}`}>
```

Comment:
Will this clash with the injected footer?

Reply:
no, the injected footer is a `div` element

Reply:
Then will this show below or above it?

Reply:
this will be shown below the post content but above the injected footer